src/assets/i18n/en.json

```diff
@@ -467,6 +467,16 @@
             "INFO": "View details"
         }
     },
+    "FILE_UPLOAD": {
```

Comment:
It is already part of ADF, no? https://github.com/Alfresco/alfresco-ng2-components/blob/development/lib/content-services/i18n/en.json#L157